src/SolutionTemplate/UnoSolutionTemplate.net6/Shared/App.xaml.cs

```diff
@@ -63,7 +48,7 @@ protected override void OnLaunched(LaunchActivatedEventArgs e)
             _window = new Window();
             _window.Activate();
 #else
-			_window = Windows.UI.Xaml.Window.Current;
+			_window = Window.Current;
```

Comment:
This is going to fail on iOS because there's a `Window` property in base classes. It needs to be fully qualified.

Reply:
```suggestion
			_window = Windows.UI.Xaml.Window.Current;
```

Reply:
Just checked and it seems like it only needs to be fully qualified in method bodies. As the type of a field it seems fine left unqualified (because it's not legal to refer to the property so there's no ambiguity I guess).

Reply:
@davidjohnoliver This line is indeed in a method body (`OnLaunched`). The other line where it's the type of a field is kept unqualified.

Reply:
oh yeah, oops :D